### ISSUE TITLE
Feature Request: Upload File parameter to override existing File

### ISSUE THREAD

**Issue by Y0ngg4n** [flymzero/webdav_client#5]
If i want to upload a file it should be possible to override this file if exists.
Maybe you can pass a boolean to the function to enable override.

**Comment by flymzero** [comment]
Yes, I'll see if the protocol itself is supported, I'll update it in the next version (not soon 😅).

**Comment by Y0ngg4n** [comment]
@flymzero thank you :)
